freq/testsettings.py

```diff
@@ -112,6 +112,7 @@
     # ^^^ bower-managed files.
 ]
 
+from freq.secretsettings import *
```

Comment:
try/except? (Perhaps not, but then add a quick comment that it is mandatory. And also say which settings you need to add (or add a 'see README' comment)).